test/system/rich_text_editor_test.rb

```diff
@@ -27,6 +31,18 @@ def setup
 
     # Make sure that image has been uploaded
     assert_equal( is_image_url_undefined, false )
+
+    find('.ple-publish').click()
+
+    assert_selector('h1', text: "My note")
+    assert_selector('#content', text: "All about this interesting stuff")
+    assert_selector('.alert-success', text: "×\nResearch note published. Get the word out on the discussion lists!")
+    page.find('.main-content img.d-print-none')['src'].should have_content 'pl.png'
```

Comment:
`Minitest::UnexpectedError:         Capybara::ElementNotFound: Unable to find css ".main-content img.d-print-none"`